src/components/templates/settingsLayout.tsx

```diff
@@ -1,16 +1,24 @@
 import React, { useEffect, useState } from "react";
 
 import { useTranslation } from "react-i18next";
-import { Outlet } from "react-router-dom";
+import { Outlet, useLocation } from "react-router-dom";
 
 import { LogoCatLarge, PageTitle } from "@components/atoms";
 import { Sidebar, TitleTopbar } from "@components/organisms";
 import { SettingsMenu } from "@components/organisms/settings";
 
 export const SettingsLayout = () => {
+	const { t: tSettings } = useTranslation("settings", { keyPrefix: "topbar" });
 	const { t } = useTranslation("global", { keyPrefix: "pageTitles" });
 	const [pageTitle, setPageTitle] = useState<string>(t("base"));
-	const { t: tSettings } = useTranslation("settings", { keyPrefix: "topbar" });
+	const { pathname } = useLocation();
+
+	const subPageTitles: Record<string, string> = {
+		"/settings": tSettings("personalSettings"),
+		"/settings/client-configuration": tSettings("personalSettings"),
+	};
+
+	const topbarTitle = subPageTitles[pathname] ? subPageTitles[pathname] : "";
```

Comment:
perhaps:
```
	const topbarTitle = pathname.contains("/settings") ? tSettings("personalSettings") : tSettings("title");
```

?

Reply:
for example: Organization page - topbar title should be _Organization_. So your option won't work.

Reply:
> perhaps:
> 
> ```
> 	const topbarTitle = pathname.contains("/settings") ? tSettings("personalSettings") : tSettings("title");
> ```
> 
> ?

changed